src/ui/screens/PermissionGrantingPrompt.tsx

```diff
@@ -28,27 +26,12 @@ function PermissionGrantingPromptScreen({skipPnPermission, allowSkip = true}: Pr
           <Caption style={globalStyles.textCenter}>
             Enable notifications to make sure you don't miss out on important events
           </Caption>
-          {error ? (
```

Comment:
Don't we want to handle errors?

Reply:
In this case this is prompting the native notification permission dialog. User can either `allow` or `don't allow` and both of the states are reflected accordingly in the UI.

Permission "Don't allow":
https://user-images.githubusercontent.com/16683923/176664107-e8939120-c804-4696-80a3-48d354c0fbd8.mp4

Permission "Allow":
https://user-images.githubusercontent.com/16683923/176664209-3e72c3de-bac2-45e9-bad6-b7edc716c96c.mp4